Library/Homebrew/shims/linux/super/cc

```diff
@@ -184,6 +184,11 @@ class Cmd
       val  = chuzzle($1) || enum.next
       path = canonical_path(val)
       args << "-L#{val}" if keep?(path) && @lset.add?(path)
+    when /^-R(.+)?/
+      # -R is not the correct way to pass rpaths to the linker
+      val  = chuzzle($1) || enum.next
+      path = canonical_path(val)
+      args << "-Wl,-rpath=" if keep?(path) && @lset.add?(path)
```

Comment:
Yeah, you're right about the lset. I've thought about this a bit more now and I think that this should just be an dumb translation of `-R` into `-Wl,-rpath=`, which will preserve current behavior. I'm thinking of also dropping the `keep?(path)` bit since it's possible that this could change certain behavior. `gcc` is often an undeclared runtime dependency of formulae built with gcc; and I think filtering paths here would change how formulae are currently built.